### PR TITLE
Feat: Create category-options schema for book entity in Contentful

### PR DESCRIPTION
**Pull request by dancol93**
Category schema for books entity in Contentful.

## TODO:
- Filter subCategories by category selected.

### CODE REVIEW

File: schemas/category-options.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
```
Comment (inigomarquinez) [inline schemas/category-options.json]: As the schema is very specific for HE project, I would suggest changing the name of the file to be more specific too. Something similar to `noordhoff-he-category-options.json`.

File: extension.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
         "name": "Default JSON Schema Path",
         "description": "Default base path to look for the JSON schemas",
         "type": "Symbol",
-        "required": true,
```
Comment (inigomarquinez) [inline extension.json]: This should be true in the production version.

File: schemas/category-options.json (hunk 1 above)
Comment (dancol93) [reply schemas/category-options.json]: You're right, following that spirit I renamed it to `noordhoff-he-book-category-options.json`

File: extension.json (hunk 2 above)
Comment (dancol93) [reply extension.json]: Thanks! Forgot to change it back 😅

**Pull request merged**